File: week1/CodeKata1.py
def getMyObject():
    // This function returns a proper object 99.9% of the time
    ...

def main():
    myVar = getMyObject();
    if myVar is None:
        // We have to check the var because 0.01% of the time it fails
        return;
    else:
        doSomething(myVar);

File: week1/CodeKata1.js
function getMyObject() {
    // This function returns a proper object 99.9% of the time
    ...
}

function main() {
    var myVar = getMyObject();
    if (!myVar) {
        // We have to check the var because 0.01% of the time it fails
        return;
    } else {
        doSomething(myVar);
    }
}
